gaphor/UML/states/propertypages.py

```diff
@@ -11,6 +14,9 @@
     handler_blocking,
     new_resource_builder,
 )
+from gaphor.UML.states.state import StateItem
+
+from .statemachine import StateMachineItem
```

Comment:
To stay consistent with the other import statements, should we use absolute imports here?

Reply:
Yes it should. I suppose VSCode tried to help me by importing this item